server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/Analysis.java

```diff
@@ -666,4 +666,27 @@ public Map<String, List<SortItem>> getDeviceToSortItems() {
   public void setDeviceToSortItems(Map<String, List<SortItem>> deviceToSortItems) {
     this.deviceToSortItems = deviceToSortItems;
   }
+
+  /////////////////////////////////////////////////////////////////////////////////////////////////
+  // Logical View Analysis
+  /////////////////////////////////////////////////////////////////////////////////////////////////
+  private boolean hasViewsInQuery = false;
+
+  public void setHasViewsInQuery(boolean hasViewsInQuery) {
+    this.hasViewsInQuery = hasViewsInQuery;
+  }
+
+  public boolean hasViewsInQuery() {
+    return this.hasViewsInQuery;
+  }
+
+  private List<Pair<Expression, String>> outputExpressions = null;
```

Comment:
move the attribute to the upper place of this class to stay with other attributes.

Reply:
fixed.